app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1616,7 +1617,7 @@ class BrowserTabViewModel(
     }
 
     suspend fun refreshCta(locale: Locale = Locale.getDefault()): Cta? {
-        if (currentGlobalLayoutState() is Browser) {
+        if (currentGlobalLayoutState() is Browser && currentCtaViewState().cta !is DaxDialog) {
```

Comment:
Added this because there is a scenario where the user agent change kicks in and sends 3 pixels saying the CTA was shown. This avoids showing a CTA if there is already one being shown.

Note: This only applies to in browser CTAs so if the CTA is different than DaxDialog we can then show it (coming from the home screen to the browser for example)

Reply:
Ended up removing this!